lib/components/Keyboard/KeyboardInput/KeyboardAccessoryView.tsx

```diff
@@ -63,10 +54,8 @@ export type KeyboardAccessoryViewProps = kbTrackingViewProps & {
    */
   onKeyboardResigned?: () => void;
   /**
-   * iOS only.
-   * The scrolling behavior, use KeyboardAccessoryView.iosScrollBehaviors.X where X is:
-   * NONE, SCROLL_TO_BOTTOM_INVERTED_ONLY or FIXED_OFFSET
-   * default: FIXED_OFFSET
+   * @deprecated
+   * Please use 'scrollBehavior' prop instead and take values from 'KeyboardTrackingView.scrollBehaviors' (instead of KeyboardAccessoryView.iosScrollBehaviors)
```

Comment:
I think it should still be `KeyboardAccessoryView.iosScrollBehaviors`, so the user will not need to import `KeyboardTrackingView` as well.
But probably change this to `scrollBehaviors` as well.

Reply:
The comment is wrong, I'll fix this